examples/google-esp-integration/with_sdk_v3/README.md

```diff
@@ -8,15 +8,34 @@ For an example application without using the UID2 SDK, see [Server-Only UID2 Int
 
 ## Build and Run the Example Application
 
-The easiest way to try the example is to use the following Docker Build command:
+### Using the VS Code Debugger
+
+The easiest way to try the example is to do the following:
+1. Open this repo in VS Code
+1. Create a `.env` file in this folder and populate `UID2_API_KEY` and `UID2_CLIENT_SECRET`:
+    ```
+    UID2_BASE_URL=http://localhost:8080
+    UID2_API_KEY=<your-integ-API-key>
+    UID2_CLIENT_SECRET=<your-integ-client-secret>
+    AD_TAG_URL=https://pubads.g.doubleclick.net/gampad/ads?iu=/21775744923/extrernal/adx-test-tag&tfcd=0&npa=0&sz=640x480&gdfp_req=1&output=vast&unviewed_position_start=1&env=vp&impl=s&correlator=
+    UID2_JS_SDK_URL=http://localhost:9091/uid2-sdk.js
+    ```
+1. Click the Run and Debug tab or hit `Crtl+Shift+D`
+1. Select `Launch ESP (Chrome)` from the configuration dropdown
+1. Click `Start Debugging` or hit F5
+
+### Running the Docker commands manually
+
+The other way to try the example is to use the following Docker Build command. First, open this folder in your terminal, then run the following:
 
 ```
 docker build . -t uid2-esp-standard
-docker run -it --rm -p 3000:3000 \
-    -e UID2_BASE_URL="https://operator-integ.uidapi.com" \
-    -e UID2_API_KEY="<your-integ-API-key>" \
-    -e UID2_CLIENT_SECRET="<your-integ-client-secret>" \
-    -e AD_TAG_URL="<your-IMA-ad-tag-url>" \
```

Comment:
This was giving me `docker: invalid reference format.` but the backticks work for me.